### PR TITLE
Fixing broken tests

### PR DESCRIPTION
**Pull request by burtonemily**

### CODE REVIEW

File: tests/automation/setup/test_user.ts [hunk 1]
```diff
@@ -1,34 +1,39 @@
...
+  sessionid:
+    '059e36f9bb03ac4e1ff9ff7d4aedd791d1f412770c6cd2796b855af12a7c7ea000',
   recoveryPhrase:
     'tuition remedy sieve ivory wiggle gemstone hawk optical issued damp selfish fewest damp',
 };
 
 export const userB = {
   userName: 'Bob',
-  sessionid: '05ead902602c1f7b59aaa3932b9cdefe8ee4075c03b09811d31bc9b122cb292d0c',
+  sessionid:
```
Comment (Bilb) [inline tests/automation/setup/test_user.ts:11]: Looks like this test_user is globally unused, to remove

File: tests/automation/utilities/utils.ts [hunk 2]
```diff
@@ -295,60 +295,42 @@ export async function hasElementBeenDeleted(
...
+      'control-message',
+      maxWait,
+      text,
+    );
+    return false; // Text message was found
   } catch (e) {
-    throw new Error('Element not defined');
+    return true; // Text message doesn't exist or wasn't found in time
   }
 }
```
Comment (Bilb) [inline tests/automation/utilities/utils.ts:327]: this function is still not doing what its name suggests

File: tests/automation/utilities/utils.ts [hunk 3]
```diff
@@ -295,60 +295,42 @@ export async function hasElementBeenDeleted(
...
-    await waitForElement(window, strategy, selector, maxWait);
-    throw new Error(fakeError);
-  } catch (e) {
-    if (e.message === fakeError) {
-      throw e;
-    }
-  }
-  console.info(`${selector} has not been found, congrats`);
-}
+    await waitForElement(window, strategy, selector, maxWait, text);
```
Comment (Bilb) [inline tests/automation/utilities/utils.ts:302]: if you catch all the exceptions and don't throw anything new, your test will pass, which in this case you don't want

File: tests/automation/message_requests.spec.ts [hunk 4]
```diff
@@ -16,79 +16,123 @@ test.beforeEach(beforeAllClean);
 // test.afterEach(() => forceCloseAllWindows(windows));
 // Open two windows and log into 2 separate accounts
 test.describe('Message requests', () => {
-  sessionTestTwoWindows('Message requests accept', async ([windowA, windowB]) => {
-    const [userA, userB] = await Promise.all([newUser(windowA, 'Alice'), newUser(windowB, 'Bob')]);
```
Comment (Bilb) [inline tests/automation/message_requests.spec.ts:20]: any reason we use test.describe for those tests only? Maybe we should just have the sessionTestTwoWindows direclty at the root of the file

**Review by Bilb — CHANGES_REQUESTED** [review]

File: tests/automation/message_requests.spec.ts (hunk 4 above)
Comment (burtonemily) [reply tests/automation/message_requests.spec.ts:20]: I think this was initially to run all the Message requests tests at once but i don't think the `describe` function works correctly

**Comment by burtonemily** [comment]
Fixed all the requested changes @Bilb

**Pull request closed without merging**